general-superstaq/general_superstaq/service.py

```diff
@@ -1,14 +1,17 @@
 import os
-from typing import Any, Dict, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
 
-import general_superstaq as gss
+import qubovert as qv
 
+if TYPE_CHECKING:
+    from general_superstaq import superstaq_client
 
-class UserConfig:
+
+class Service:
     """This class contains all the user configurations that are used to operate Superstaq."""
 
-    def __init__(self, client: gss.superstaq_client._SuperstaqClient):
-        """Initializes the `UserConfig` class.
+    def __init__(self, client: "superstaq_client._SuperstaqClient"):
```

Comment:
```suggestion
    def __init__(self, client: gss.superstaq_client._SuperstaqClient):
```

Works for me. Could be something wrong with your local machine.

Reply:
Seems to be a local issue - here is the following error I get when I use that line:

`AttributeError: partially initialized module 'general_superstaq' has no attribute 'superstaq_client' (most likely due to a circular import)`